frontend/src/components/Utils/time.js

```diff
@@ -93,15 +94,28 @@ export function getDateBarebones(msTimestamp, timezone=null) {
 }
 
 /**
- * Get a time in 24-hour ('HH:mm') form. 
+ * Get a time in 24-hour ('HH:mm') format. 
  * 
  * @param {string} msTimestamp Timestamp, in milliseconds since epoch.
  * @param {string} timezone The timezone which the string should be returned in.
- * @returns {string} The time in 24-hour (HH:mm) form.   
+ * @returns {string} The time in 24-hour (HH:mm) format.   
  */
 export function get24hTime(msTimestamp, timezone=null) {
   if (timezone === null) {
     return moment.tz(parseFloat(msTimestamp), '').format('HH:mm');
   }
   return moment.tz(parseFloat(msTimestamp), timezone).format('HH:mm');
+}
+
+/**
+ * Get a Firebase Timestamp object for time.
+ *
+ * @param {string} time The time in 'HH:mm' format.
+ * @param {string} date The date in 'YYYY-MM-DD' format.
+ * @param {string} tz The timezone in which the date takes place.
+ * @returns {firestore.Timestamp} Firestore timestamp object at the same time. 
+ */
+export function getFirebaseTime(time, date, tz) {
```

Comment:
Nit (optional): You may want to be more specific in function name. You could try something along the lines of `firebaseTimestampFromISOTime`. No pressure if you like what you have though.